src/scss/components/box/_index.scss

```diff
@@ -0,0 +1,3 @@
+@import "mixins";
+
+// TODO: convert remaining LESS styles
```

Comment:
FYI: Light DOM styles for the Box component have yet to be migrated. This will happen in a future PR.

Reply:
Changes to Box mixins will be handled by #588. This PR should not have any changes to this file.